javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -421,13 +470,46 @@ class Driver extends webdriver.WebDriver {
    */
   setFileDetector() {
   }
+
+  /**
+   * Get the context that is currently in effect.
+   *
+   * @return {Context} Current context.
+   */
+  get context() {
```

Comment:
I originally coded this in, but then realised it might be better to let it fail with an “unknown command” error instead. Though I guess this can be confusing to users because they might not realise it’s specific to Marionette.

This comes down to stylistic preferences. Would you like to check if we’re talking to a Marionette remote and throw or leave it as-is?